Sources/AppStoreConnectCLI/Model/Build.swift

```diff
@@ -104,7 +87,7 @@ extension Build: TableInfoProvider {
     }
 
     var tableRow: [CustomStringConvertible]  {
-        return [
+        let row = [
```

Comment:
Do we need this temp var row?

Reply:
I was just attempting to make the code clearer / read nicer. In all honesty it's probably optimized out by the compiler. It's not strictly necessary, no.

Reply:
It's just other places we are returning without taking it in a temp var row..didn't look consistent to me

Reply:
I not sure this level of consistency is worth worrying about too much
Some of these rows we do a line by line nil coalesce and others we do a `compactMap`
It's a simple computed variable that needs to return a valid row